bin/test

```diff
@@ -8,3 +8,24 @@ xcrun xcodebuild \
   test \
   | xcpretty --color
 
+xcrun xcodebuild \
+  -workspace Argo.xcworkspace \
+  -scheme Argo-iOS \
+  -destination "platform=iOS Simulator,name=iPhone 6" \
+  test \
+  | xcpretty --color
+
+xcrun xcodebuild \
+  -workspace Argo.xcworkspace \
+  -scheme Argo-watchOS \
+  -destination "generic/platform=watchOS" \
+  CODE_SIGNING_REQUIRED=NO \
+  CODE_SIGN_IDENTITY="" \
```

Comment:
so is the watch one special since we can't run tests on it. This is just making sure it builds?